aiida_cp2k/calculations/__init__.py

```diff
@@ -270,6 +286,16 @@ def prepare_for_submission(self, folder):
                 conflicting_keys=["COORDINATE"],
             )
 
+        # Craeate input trajectory files
+        if "trajectory" in self.inputs:
+            self._write_trajectories(
+                self.inputs.structure,
```

Comment:
Why do you need the structure here? Isn't all the information available in the trajectory itself?

Reply:
no the trajectory does not carry information on the initial magnetization guess or, if needed in the future, ghost atoms

Reply:
Despite @cpignedoli didn't like my idea, he removed this argument in [06a33f3](https://github.com/aiidateam/aiida-cp2k/pull/207/commits/06a33f311c627228536f5e8963aa56da173eeabc) 😆